src/MockManager.php

```diff
@@ -305,6 +305,8 @@ protected static function mock(
 		string $mockKey,
 		self::InternalMockCallback $callback,
 	): void {
+    /* HH_FIXME[2049] This function is not in any hhi */
+    /* HH_FIXME[4107] This function is not in any hhi */
```

Comment:
Should we just create the `.hhi` files?

Reply:
That would work for 3.30, but cause issues in hhvm 4.2something. I'll do something clever, once we reach a release with fb_intercept2().